external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -485,4 +485,32 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationSuite with SharedSQLCo
     }
     assert(df2.collect.toSet === expectedResult)
   }
+
+  test("query JDBC option") {
```

Comment:
How did you run this test suite in docker?

Reply:
@gatorsmile I did the following steps to run the oracle tests.
1) I pulled  christophesurmont/oracle-xe-11g 
2) I set the environment variable ORACLE_DOCKER_IMAGE_NAME to the above image name
3) I put the jdbc driver in maven directory
4) Then ran the test